skins/cat17/src/app/tests/view_handlers/test_section_info.js

```diff
@@ -422,3 +423,146 @@ test( 'Opened longtext are shut', function ( t ) {
 
 	t.end();
 } );
+
+test( 'Donor type info without entered data indicated correctly', function ( t ) {
```

Comment:
When looking at `DonorTypeSectionInfo` I was wondering why there are multiple classes(?) in a single file. Now when looking at this test I am wondering this once again.

In case of the production code it makes finding a specific class a bit harder, though it can make working on those related classes (especially during creation) easier and perhaps has other benefits. Based on the info I have I would still go for one class per file.

In case of the tests there are multiple functions that test a specific production class in the top level scope of the file, together with functions that test other classes, a downside absent for the production code here. This issue is compounded when extracting code from the test functions (either for sharing or single responsibility and single level of abstraction).